internal/description/description.go

```diff
@@ -45,6 +45,8 @@ const (
 	CreateDataLake           = "Create a new data lake."
 	UpdateDataLake           = "Update a data lake."
 	DeleteDataLake           = "Delete a data lake."
+	Certs                    = "Manage x509 certificates for your database users."
```

Comment:
```suggestion
	Certs                    = "Customer managed x509 certificates for your database users."
```

Reply:
The rest of the descriptions all list the actions the command can do, does "Manage your own x509 certificates ..." work? Or "Manage customer x509 certificates"

Reply:
>Manage customer x509 certificates

Works for me, just trying to clarify this, doesn't matter if the other commands mention is customer related as `mongocli security customerCerts --help` will only show this help text